paparazzi/paparazzi-gradle-plugin/src/test/projects/build-class/src/test/java/app/cash/paparazzi/plugin/test/BuildClassTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paparazzi.plugin.test
+
+import android.os.Build
+import app.cash.paparazzi.Paparazzi
+import org.junit.Rule
+import org.junit.Test
+
+class BuildClassTest {
+  @get:Rule
+  val paparazzi = Paparazzi()
+
+  @Test
+  fun verifyFields() {
+    println(Build.MODEL)
+    assert(Build.ID == "TPB4.220624.004")
```

Comment:
@geoff-powell you used the built-in `assert` here instead of `assertEquals`, as far as I know this depends on `-ea`, so these might be not executed all the time giving a false positive test success. Was this intentional?